server/manager/src/main/java/org/apache/accumulo/manager/compaction/queue/CompactionJobPriorityQueue.java

```diff
@@ -136,18 +143,35 @@ public synchronized boolean add(TabletMetadata tabletMetadata, Collection<Compac
     return true;
   }
 
+  public long getSize() {
+    return maxSize;
+  }
+
+  public long getRejectedJobs() {
+    return rejectedJobs.get();
+  }
+
+  public long getDequeuedJobs() {
+    return dequeuedJobs.get();
+  }
+
+  public long getQueuedJobs() {
+    return queuedJobs.get();
+  }
```

Comment:
For this could we ask the queue for its size? Then don't need to worry about keeping the atomic long up to date.

```suggestion
  public synchronized long getQueuedJobs() {
    return jobQueue.size();
  }
```